src/SourceBuild/content/repo-projects/installer-deps.props

```diff
@@ -0,0 +1,24 @@
+<Project>
+
+  <ItemGroup>
+    <RepositoryReference Include="arcade" />
+    <RepositoryReference Include="aspire" />
+    <RepositoryReference Include="aspnetcore" />
+    <RepositoryReference Include="command-line-api" />
+    <RepositoryReference Include="deployment-tools" />
+    <RepositoryReference Include="emsdk" />
+    <RepositoryReference Include="fsharp" />
+    <RepositoryReference Include="msbuild" />
+    <RepositoryReference Include="nuget-client" />
+    <RepositoryReference Include="roslyn" />
+    <RepositoryReference Include="runtime" />
+    <RepositoryReference Include="scenario-tests" />
```

Comment:
Why is this declared as a dependency on installer?

Reply:
Yeah, it's really a dependency of dotnet.proj

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/edebea74ce5d93ed82140d750bc729322b03d233.